backend/controller/sql/queries.sql

```diff
@@ -798,3 +738,13 @@ UPDATE topic_subscriptions
 SET state = 'idle'
 WHERE name = @name::TEXT
       AND module_id = (SELECT id FROM module);
+
+-- name: CreateCronJob :exec
+SELECT create_cron_job(
```

Comment:
The Controller should be creating cron jobs through the cron DAL...

Reply:
If this is due to transactions, we need to refactor how transactions work, not duplicate this.

Reply:
Done

Reply:
I only needed to add a `Tx()` getter method to get the underlying `pgx.Tx` object in `sql/conn.go`. That made it possible to use the sqlc-generated `WithTx()`, so now all the child DALs' queries are wrapped in the same parent transaction. We only need to use that when transacting through multiple DALs. The rest of the code stays the same.